evennia/utils/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.dispatch import Signal
+
+
+ACCOUNT_CREATE = Signal(providing_args=['session', ])
```

Comment:
These need at least a line of comment to explain exactly when it fires (is it before/after the account object is created, for example? Which hooks have fired already at this point?). Same goes for the other signals below.